openapi/components/paths/users.yaml

```diff
@@ -73,7 +73,7 @@ paths:
       tags:
         - users
       x-codeSamples:
-        $ref: "../codeSamples/users.yaml#/~1users~1{userId}/get"
+        $ref: '../codeSamples/users.yaml#/~1users~1{userId}/get'
```

Comment:
we use double quotes when possible :)

Reply:
Actually I think single quote is mostly used?

Let's pause on this one for now, I need to check with the linter.

Reply:
Double quotes got automatically replaced on file open by Stoplight multiple times now.

Reply:
Moved to #200 for linting quotes , marking conversation as resolved.